src/gui/mzroll/tabledockwidget.h

```diff
@@ -10,7 +10,11 @@
 #include "numeric_treewidgetitem.h"
 #include "QHistogramSlider.h"
 #include "saveJson.h"
-#include "jsonReports.h";
+#include "jsonReports.h"
+#include "polyFit.h"
```

Comment:
@TanmayGoyal Why do we need to include alignment in tabledockwidget?

Reply:
@sahil21 We need to include one of the header files for alignment algorithms ( which is currently polyfit ) as tabledockwidget uses it. We can however remove the other 2 header files.